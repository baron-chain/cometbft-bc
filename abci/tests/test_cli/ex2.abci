# Transaction Validation Tests
check_tx 0x00
>> code: OK

check_tx 0xff
>> code: OK

# Initial Transaction Delivery
deliver_tx 0x00
>> code: OK

# Additional Transaction Validation
check_tx 0x00
>> code: OK

# Multiple Transaction Deliveries
deliver_tx 0x01
>> code: OK

deliver_tx 0x04
>> code: OK

# State Query
info
>> code: OK
>> data: {"size":3}
>> data.hex: 7B2273697A65223A337D
