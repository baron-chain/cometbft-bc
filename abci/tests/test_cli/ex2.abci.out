# Initial Transaction Validation
check_tx 0x00
>> code: OK

# Boundary Value Check
check_tx 0xff
>> code: OK

# First Transaction Delivery
deliver_tx 0x00
>> code: OK

# Repeated Transaction Validation
check_tx 0x00
>> code: OK

# Sequential Transaction Deliveries
deliver_tx 0x01
>> code: OK

deliver_tx 0x04
>> code: OK

# Final State Query
info 
>> code: OK
>> data: {"size":3}
>> data.hex: 7B2273697A65223A337D
